.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ _db_docker_config: &db_docker_config
   - image: cimg/openjdk:17.0
     environment:
       POSTGRES_DB: interventions_ci
-  - image: circleci/postgres:10-alpine-ram
+  - image: postgres:14.2
```

Comment:
🙋‍♂️ `cimg/postgres:14.2` replaced the older `circleci/postgres` images

Reply:
Yeah, what confused me is when updating it to `image: cimg/postgres:14.2` circleci fails because the docker pull command has changed from what was "docker pull circleci/postgres:10-alpine-ram" to "docker pull postgres:14.2" so by using the "cimg/" it was pulling an image which didn't exist within docker hub. Confusing because the docs even say to use cimg/postgres:14.2 but it didn't work